sdk/typescript/runtime/main.go

```diff
@@ -60,6 +60,9 @@ const (
 
 // ModuleRuntime returns a container with the node or bun entrypoint ready to be called.
 func (t *TypeScriptSdk) ModuleRuntime(ctx context.Context, modSource *ModuleSource, introspectionJson string) (*Container, error) {
+	ctx, span := Tracer().Start(ctx, "ModuleRuntime")
+	defer span.End()
```

Comment:
I don't think we should add these per-function spans since Dagger will automatically do that for you at the function level. I wouldn't want folks to see this pattern and think they need to do it themselves. Manually added spans should only be for custom parallelism or grouping that the user wants to add on top.

Reply:
Done